lightning/src/ln/channel.rs

```diff
@@ -1656,6 +1657,54 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 		cmp::min(self.value_to_self_msat as i64 - self.get_outbound_pending_htlc_stats().1 as i64, 0) as u64)
 	}
 
+	fn commit_tx_fee_outbound_htlc(&self) -> u64 {
+		if !self.channel_outbound {
+			return 0;
+		} else {
+			let mut their_acked_htlcs = self.pending_inbound_htlcs.len() as u64;
+
+			for ref htlc in self.pending_outbound_htlcs.iter() {
+				match htlc.state {
+					OutboundHTLCState::Committed => their_acked_htlcs +=1,
+					OutboundHTLCState::RemoteRemoved {..} => their_acked_htlcs +=1,
+					OutboundHTLCState::LocalAnnounced {..} => their_acked_htlcs += 1,
+					_ => {},
+				}
+			}
+
+			for ref htlc in self.holding_cell_htlc_updates.iter() {
+				match htlc {
+					&&HTLCUpdateAwaitingACK::AddHTLC { .. } => their_acked_htlcs += 1,
+					_ => {},
+				}
+			}
+
+			return self.feerate_per_kw * (COMMITMENT_TX_BASE_WEIGHT + their_acked_htlcs * COMMITMENT_TX_WEIGHT_PER_HTLC);
+		}
+	}
+
+	fn commit_tx_fee_inbound_htlc(&self) -> u64 {
```

Comment:
I think this name could be more clear - something like current_tx_fee_in_next_remote_comimtment_tx or something (luckily we don't pay by the character).

Reply:
Lmk if new name isn't clear